ny-portal/groupBy.d.ts

```diff
@@ -0,0 +1,11 @@
+interface ObjectConstructor {
```

Comment:
Kanskje skrive noe om hvorfor dette er her?

```suggestion
// TODO: Fjern denne når vi har oppdatert til en Typescript-versjon som støtter Object.groupBy
interface ObjectConstructor {
```